gitnamehere/point-of-sale-group-project
# Theming fix

This PR fixes the delay in applying the theme in slower devices
Theming has been changed from using a js file and setting each element with the classname with it respective style, to a server generated css file similar to how dynamic html was done in class

Testing:
* `npm run start:dev`
* go to the pos and verify theming works
* go to every page with theming and verify that there is no flickering/delay when the theme is being applied
